app/views/unloading_permission/components/table_2/bottom_right_block.scala.xml

```diff
@@ -9,10 +9,18 @@
         <fo:table-body>
             <fo:table-row>
                 <fo:table-cell padding-left="2mm">
-                    @permission.goodsItems.head.specialMentions.map(specialMention => item.special_mentions(specialMention))
+                    @if(permission.goodsItems.head.specialMentions.nonEmpty) {
+                        @permission.goodsItems.head.specialMentions.map(specialMention => item.special_mentions(specialMention))
+                    } else {
+                        <fo:block/>
+                    }
                 </fo:table-cell>
                 <fo:table-cell padding-left="3mm">
-                    @permission.goodsItems.head.producedDocuments.map(producedDocument => item.produced_document(producedDocument))
+                    @if(permission.goodsItems.head.producedDocuments.nonEmpty) {
+                        @permission.goodsItems.head.producedDocuments.map(producedDocument => item.produced_document(producedDocument))
```

Comment:
Will this always just be the head, if so can we guarantee that the head is going to be the correct value?

Reply:
We're within an if here that checks and makes sure only a single item exists (this is the first page, we only allow single item movements to go on 1st page). Should be confident it exists because GoodsItem is a NonEmptyList